### DISCUSSION TITLE
Interest in a SwiftNIO broker?

### DISCUSSION THREAD

**Discussion by sbeitzel** [adam-fowler/mqtt-nio#127] [Q&A]
Does a SwiftNIO broker implementation exist already? And if not, is there interest in one? I found [this Swift implementation](https://github.com/ckrey/mqttswift) of a bare-bones broker (no web socket, e.g.) but otherwise radio silence. What do you think?

**Comment by adam-fowler** [comment]
I don't know of any swift-nio server/broker implementations. From a use point of view I imagine there is less interest in one given the server is a standalone entity and does not interact with any other Swift code. What would a Swift implementation bring to the table that isn't provided by the implementations already out there?
